src/data/ab-attrs/reduce-sleep-duration-ab-attr.ts

```diff
@@ -1,20 +1,21 @@
 import type { Pokemon } from "#app/field/pokemon";
 import type { NumberHolder } from "#app/utils";
-import type { StatusEffect } from "#enums/status-effect";
+import { StatusEffect } from "#enums/status-effect";
 import { AbAttr } from "./ab-attr";
 
 /**
- * Used by Early Bird, makes the pokemon wake up faster
+ * /**
+ * This attribute reduces the duration of sleep by half and is used by the ability Early Bird.
+ * Early Bird halves the sleep duration. When the Pokémon is put to sleep, the number of turns it will remain asleep is preset, between 1 and 5 (the range depends on the generation).
```

Comment:
```suggestion
 * Early Bird halves the sleep duration. When the Pokémon is put to sleep, the number of turns it will remain asleep is preset, between 1 and 3.
```
nit: We probably don't need to mention previous generations